File: 1266-minimum-time-visiting-all-points/1266-minimum-time-visiting-all-points.go
func minTimeToVisitAllPoints(points [][]int) int {
    ans := 0
    for i := 0; i < len(points)-1; i++ {
        xDiff := abs(points[i+1][0] - points[i][0])
        yDiff := abs(points[i+1][1] - points[i][1])
        ans += max(xDiff, yDiff)
    }    
    return ans
}

func abs(x int) int {
    if x < 0 {
        x *= -1
    }
    return x
}

File: 1266-minimum-time-visiting-all-points/1266-minimum-time-visiting-all-points.cpp
class Solution {
public:
    int minTimeToVisitAllPoints(vector<vector<int>>& points) {
        int n = points.size();
        int ans = 0;
        for(int i = 0; i < n-1; i++) {
            int xdiff = abs(points[i+1][0] - points[i][0]);
            int ydiff = abs(points[i+1][1] - points[i][1]);
            ans += max(xdiff, ydiff);
        }       

        return ans;
    }
};